framework/src/problems/FEProblemBase.C

```diff
@@ -1735,7 +1735,7 @@ void
 FEProblemBase::reinitElemPhys(const Elem * elem,
                               const std::vector<Point> & phys_points_in_elem,
                               THREAD_ID tid,
-                              bool suppress_displaced_init)
```

Comment:
This isn't used in the DisplacedProblem method either - can we remove it? Or are there other applications that still pass something in here?